src/main/thrift/parquet.thrift

```diff
@@ -833,6 +840,15 @@ struct ColumnMetaData {
    * filter pushdown.
    */
   16: optional SizeStatistics size_statistics;
+
+  /* True if all pages in this column chunk are dictionary encoded */
+  17: optional bool is_fully_dict_encoded; 
```

Comment:
could we keep this in a separate PR and handle the conflict separately.  In my mind at least schema index is a less controversial change.

Reply:
Done.